app/src/main/java/org/simple/clinic/addidtopatient/searchforpatient/AddIdToPatientSearchScreen.kt

```diff
@@ -0,0 +1,107 @@
+package org.simple.clinic.addidtopatient.searchforpatient
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.view.inputmethod.EditorInfo
+import android.widget.EditText
+import android.widget.ImageButton
+import android.widget.RelativeLayout
+import com.google.android.material.textfield.TextInputLayout
+import com.jakewharton.rxbinding2.view.RxView
+import com.jakewharton.rxbinding2.widget.RxTextView
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers.mainThread
+import io.reactivex.schedulers.Schedulers.io
+import kotterknife.bindView
+import org.simple.clinic.R
+import org.simple.clinic.activity.TheActivity
+import org.simple.clinic.router.screen.ScreenRouter
+import org.simple.clinic.search.results.PatientSearchResultsScreenKey
+import org.simple.clinic.widgets.PrimarySolidButtonWithFrame
+import org.simple.clinic.widgets.ScreenDestroyed
+import org.simple.clinic.widgets.showKeyboard
+import javax.inject.Inject
+
+class AddIdToPatientSearchScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context, attrs) {
```

Comment:
This screen and its corresponding controller (`AddIdToSearchScreenController`) is (currently) a clone of `PatientSearchScreen` and its controller. This is because they mostly have the same behaviour, but there are some differences in the UI.

This screen is not being used in this PR, and will be hooked up to the UI in the next PR. This next PR will also introduce the new layout for this screen and the changes that will compete this feature.

Reply:
> This screen and its corresponding controller (`AddIdToSearchScreenController`) is (currently) a clone of `PatientSearchScreen` and its controller. This is because they mostly have the same behaviour, but there are some differences in the UI.
> 
> This screen is not being used in this PR, and will be hooked up to the UI in the next PR. This next PR will also introduce the new layout for this screen and the changes that will complete this feature.

@rakshakhegde